ui/src/ducks/app/monitoring.js

```diff
@@ -1,32 +1,35 @@
-import { put, takeEvery, select, call } from 'redux-saga/effects';
+import { put, takeEvery, call } from 'redux-saga/effects';
 import { getAlerts, getClusterStatus } from '../../services/prometheus/api';
+import { intl } from '../../translations/IntlGlobalProvider';
```

Comment:
You shouldn't translate messages in the Redux state, as it hinders reproducibility. It's a presentational component's responsibility actually, IMO.

Reply:
Good point. By the way core-ui components does not support i18n yet such as Notifications so we have to do it in  sagas

Reply:
Wrap them in _container_ components then ;)

IMHO, the Redux state is not about user-facing data, it's about the application state. Your components are responsible for displaying meaningful information, derived from this application state. So, instead of storing status labels in there, just compute it from the status of components (UP vs. DOWN), and translate it on-the-fly.